generators/server/templates/_pom.xml

```diff
@@ -919,6 +919,14 @@
                             <artifactId>mapstruct-processor</artifactId>
                             <version>${mapstruct.version}</version>
                         </path>
+<%_ if (databaseType === 'sql') { _%>
```

Comment:
As with the mapstruct processor, it's much simpler, to unconditionally generate all the JPA static metamodel classes, in the server generator, instead of adding, when the first entity is created with filtering.
 We can fix this, but I think, it became a bit too complicated.